projects/plugins/backup/src/js/components/Admin.js

```diff
@@ -184,9 +152,9 @@ const Admin = () => {
 							<span>{ __( 'Consider upgrading to real-time protection', 'jetpack-backup' ) }</span>
 						</a>
 					) }
-				</div>
-				<div class="lg-col-span-1 md-col-span-1 sm-col-span-0"></div>
-				<div class="lg-col-span-5 md-col-span-3 sm-col-span-4">
+				</Col>
+				<Col lg={ 1 } md={ 2 } sm={ 0 } />
```

Comment:
This looks like the source of the break point difference
```suggestion
				<Col lg={ 1 } md={ 1 } sm={ 0 } />
```